node/src/actors/chain_manager/mod.rs

```diff
@@ -671,6 +669,64 @@ impl ChainManager {
         }
     }
 
+    /// Create a superblock and broadcast it
```

Comment:
```suggestion
    /// Create a superblock, sign a superblock vote and broadcast it
```

Maybe we could add a FIXME to avoid signing superblocks in case the identity is not in the signing committee